cms/templates/widgets/deprecated-course-key-warning.html

```diff
@@ -0,0 +1,16 @@
+<%!
+from django.conf import settings
+from django.utils.translation import ugettext as _
+is_enabled = settings.FEATURES.get('SHOW_STUDIO_WARNING_FOR_DEPRECATED_COURSE_KEYS', False)
+%>
+
+%if is_enabled and context_course and (context_course.id.deprecated or True):
```

Comment:
TODO: Remove the `True` short-circuit prior to merge; it's for ease of testing.

@ormsbee Is there an easy way to spin up a slash-separated course in a new devstack? (I haven't tried this in _years_...)

Reply:
Buried in the configuration YAML somewhere is the list of modulestores. I think first one is default, and you have to swap them and restart Studio.